src/store/Settings.js

```diff
@@ -0,0 +1,33 @@
+const state = {
+    timezones: [],
+    languages: ["English", "Spanish", "French"]
```

Comment:
Need each language to have a key:
```suggestion
    languages: {
        "en-US": {
            title: "English (United States)"
        },
        "es-DO": {
            label: "Spanish (Dominican Republic)",
            title: "Español (República Dominicana"
        },
        "fr-FR": {
            label: "French (France)",
            title: "Français (France)"
        }
    }
```